plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestingH2JdbcModule.java

```diff
@@ -72,10 +72,15 @@ public ConnectionFactory getConnectionFactory(BaseJdbcConfig config, CredentialP
     public static Map<String, String> createProperties()
     {
         return ImmutableMap.<String, String>builder()
-                .put("connection-url", format("jdbc:h2:mem:test%s;DB_CLOSE_DELAY=-1", System.nanoTime() + ThreadLocalRandom.current().nextLong()))
+                .put("connection-url", getConnectionUrl())
                 .buildOrThrow();
     }
 
+    public static String getConnectionUrl()
```

Comment:
`make...` or `new....`? especially that the result is randomized.

Reply:
BTW make intro of this method a separate commit so that you can apply it to other places like https://github.com/trinodb/trino/blob/e50edc163be6b7e150f78abfc39d092495abd87f/plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestLazyConnectionFactory.java#L48

Reply:
or maybe just roll this back for now?

Reply:
I just renamed it for now.